### ISSUE TITLE
Multiple glob patterns for the same group of reviewers

### ISSUE THREAD

**Issue by sergiubologa** [necojackarc/auto-request-review#74]
Hello! Thanks for the nice tool, it's really helpful!

I'd like to ask if the following use case is supported: I want to define multiple glob patterns and all of them to have the same group assigned. This is helpful when I can't combine multiple glob patterns into a single one. For example:
```yml
files:
  'main/!(subFolder)**/*':
  'main/!(subFolder)*':
    - reviewersGroup
```

Thanks.

**Comment by necojackarc** [comment]
Could you try the following? Internally, the glob patterns will be passed on to [minimatch](https://github.com/isaacs/minimatch), which I believe supports this syntax 😃 

```yaml
files:
  '{main/!(subFolder)**/*,main/!(subFolder)*}':
    - reviewersGroup
```

**Comment by sergiubologa** [comment]
Right, seems like it's working. But I haven't found this anywhere. Though, I didn't look into minimatch. 
Thanks, this solves my problem.
